sentry-log/src/logger.rs

```diff
@@ -1,47 +1,158 @@
-use crate::converters::{breadcrumb_from_record, event_from_record};
-use crate::LogIntegration;
+use log::Record;
+use sentry_core::protocol::{Breadcrumb, Event};
+
+use crate::converters::{breadcrumb_from_record, event_from_record, exception_from_record};
+
+/// The Action that Sentry should perform for a [`log::Metadata`].
+#[derive(Debug)]
+pub enum LevelFilter {
```

Comment:
Can we give this a different name so that this can be imported alongside `log::LevelFilter`?